growth/too/views.py

```diff
@@ -564,6 +564,16 @@ class PlanManualForm(ModelForm):
         validators=[validators.DataRequired()],
         default='REPLACE ME')
 
+    submitter = TextField(
+        validators=[validators.DataRequired()],
```

Comment:
Is this supposed to be the user's name? If so, we can get that from `current_user`.

Reply:
Yup!

Reply:
Yes its gone. see updated request.